Description: Fix FTBFS on macro #defines needed from sys/debug.h
  Fix build failure, we now need various debug macros from sys/debug.h
Author: Colin Ian King <colin.king@canonical.com>
Origin: ubuntu
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2020-11-04

Index: zfs-linux-0.8.4/include/spl/sys/list.h
===================================================================
--- zfs-linux-0.8.4.orig/include/spl/sys/list.h
+++ zfs-linux-0.8.4/include/spl/sys/list.h
@@ -26,6 +26,7 @@
 #define	_SPL_LIST_H
 
 #include <sys/types.h>
+#include <sys/debug.h>
 #include <linux/list.h>
 
 /*
